server/js/upload.js

```diff
@@ -0,0 +1,18 @@
+Meteor.methods({
+  parseUpload(data) {
+    check(data, Array);
+
+    for (let i = 0; i < data.length; i++) {
+      let item = data[i],
+        exists = Eboger.findOne({
+          desc: item.desc
+        });
+
+      if (!exists) {
+        Eboger.insert(item);
+      } else {
+        console.warn('Rejected. This item already exists.');
+      }
+    }
+  }
+});
```

Comment:
This looks like a good start, converting the CSV to JSON with parseUpload().

But we should use our Dataset collection instead of Eboeger.

Also aren't you missing an:

`import { Eboeger } from '../collections/eboeger.js';`

For this to work in any case? Right now you are relying on this to be imported some other place.

And ofcourse it should be:

`import { Dataset } from '../collections/dataset.js';`